sql/core/benchmarks/FilterPushdownBenchmark-jdk11-results.txt

```diff
@@ -2,669 +2,669 @@
 Pushdown for many distinct value case
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 11.0.10+9-LTS on Linux 5.4.0-1043-azure
-Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Select 0 string row (value IS NULL):      Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Parquet Vectorized                                10512          10572          58          1.5         668.4       1.0X
-Parquet Vectorized (Pushdown)                       596            621          19         26.4          37.9      17.6X
-Native ORC Vectorized                              8555           8723          97          1.8         543.9       1.2X
-Native ORC Vectorized (Pushdown)                    592            609          11         26.6          37.7      17.8X
+Parquet Vectorized                                 9788          10231         259          1.6         622.3       1.0X
+Parquet Vectorized (Pushdown)                       493            536          29         31.9          31.3      19.9X
+Native ORC Vectorized                              6487           6575         137          2.4         412.4       1.5X
+Native ORC Vectorized (Pushdown)                    436            447          14         36.1          27.7      22.4X
```

Comment:
Just a question. Does these results show Parquet's performance regression? This PR touches Parquet only and the ratios of Parquet and ORC were 17.6x and 17.8x respectively before. And, now it becomes 19.9x and 22.4. I expected `19.x and 19.x` or `22.x and 22.x` similarily.

Reply:
No. Github action runs on different machines, there is a performance difference between them.